docs/appendices/release-notes/unreleased.rst

```diff
@@ -48,7 +48,10 @@ None
 Deprecations
 ============
 
-None
+- The :ref:`gateway.expected_nodes <gateway.expected_nodes>` cluster setting
+  has been marked as deprecated and will be removed in CrateDB 5.0.
+  The :ref:`gateway.expected_data_nodes <gateway.expected_data_nodes>` must be
+  used instead.
```

Comment:
`gateway.expected_master_nodes`, `gateway.recover_after_nodes` and `gateway.recover_after_master_nodes` are also marked as deprecated, shouldn't they be mentioned as well?

Reply:
For `gateway.recover_after_nodes`, yes good catch, will fix.
As we've never documented/exposed the related *_master_* settings, I think we don't have to mention them.

Reply:
Thank you. Yes this impacts Cloud. We'll track this in Trello and update.